chain/client/src/view_client.rs

```diff
@@ -545,6 +545,8 @@ impl Handler<GetBlock> for ViewClientActor {
 
     #[perf]
     fn handle(&mut self, msg: GetBlock, _: &mut Self::Context) -> Self::Result {
+        let _timer =
+            metrics::VIEW_CLIENT_MESSAGE_TIME.with_label_values(&["GetBlock"]).start_timer();
```

Comment:
This is kinda sus.  I wonder if we could hook into actix and do the measurement at that level so we don’t have to repeat this line over and over.

Reply:
Yeah that would be nice. I just don't know if something like that exists.